### PR TITLE
feat(model): add model module

### PR DESCRIPTION
**Pull request by hewenguang**
**What this PR does / why we need it**:

add model module to klever web

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: src/pages/model/projects/_add.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import _ from 'lodash';
```
Comment (lqxu) [inline src/pages/model/projects/_add.tsx:1]: 文件名为啥要带下划线

**Review by lqxu — COMMENTED** [review]
/lgtm

**Review by grootfish — COMMENTED** [review]
/lgtm

**Pull request merged**